.github/workflows/promote-charm.yaml

```diff
@@ -37,14 +37,12 @@ jobs:
       run: |
         track=latest
 
+        # Only supports promoting charms to jammy until all Anbox Cloud
+        # charms support running on Noble.
```

Comment:
We will have something in edge after initial publishing so we can directly add promotion support

Reply:
+1, make sense to me.